testdata/project-v4/go.mod

```diff
@@ -3,7 +3,7 @@ module sigs.k8s.io/kubebuilder/testdata/project-v4
 go 1.22.0
 
 require (
-	github.com/cert-manager/cert-manager v1.16.1
+	github.com/cert-manager/cert-manager v1.16.2
```

Comment:
this one are unrelated changes.

Reply:
Yep, it was the same problem I saw here: https://github.com/kubernetes-sigs/kubebuilder/pull/4339#issuecomment-2489567278
Now that the PR mentioned in the comment has been merged I can rebase and this autogenerated changes will be gone :)